### PR TITLE
More tools

### PR DESCRIPTION
**Pull request by alkor**
New `uahistoryread` tool. Now it only supports reading raw values, other modes will be added in future versions.

Also, fixed some typos.

### CODE REVIEW

File: opcua/status_code.py [hunk 1]
```diff
@@ -663,4 +663,4 @@ def get_name_and_doc(val):
     elif val == 0x80B70000:
         return 'BadMaxConnectionsReached', 'The operation could not be finished because all available connections are in use.'
     else:
-        return 'UknownUaError', 'Unknown StatusCode value: {}'.format(val)
+        return 'UnknownUaError', 'Unknown StatusCode value: {}'.format(val)
```
Comment (oroulet) [inline opcua/status_code.py]: as the first line states this file is autogenerated :-), you need to fix generate_statuscodes.py under schemas

Comment (alkor) [inline opcua/status_code.py]: Yes, I fixed this in both places :-)

File: opcua/client.py [hunk 2]
```diff
@@ -307,3 +307,6 @@ def get_namespace_array(self):
     def get_namespace_index(self, uri):
         uries = self.get_namespace_array()
         return uries.index(uri)
+
+    def history_read(self, params):
+        return self.bclient.history_read(params)
```
Comment (oroulet) [inline opcua/client.py]: the Client class is supposed to be a very high level class and expose BinaryClient for users who want more control, so there is no need to expose the same method in Client and BinaryClient. (btw maybe we should rename bclient to binary_client to make it more discoverable)
I do not remember well the history API but I think that the history_read or better read_history method should be moved to Node class. Maybe just move read_raw_history to Node and call it read_history, if you think it makes sense

File: opcua/tools.py [hunk 3]
```diff
@@ -398,4 +400,69 @@ def uadiscover():
 
     sys.exit(0)
 
+def read_raw_history(client, node_id, startTime, endTime):
```
Comment (oroulet) [inline opcua/tools.py]: move to Node class

File: opcua/status_code.py (hunk 1 above)
Comment (oroulet) [inline opcua/status_code.py]: you allready did it!!! Sorry for the noise

File: setup.py [hunk 4]
```diff
@@ -33,6 +33,7 @@
                         'uabrowse = opcua.tools:uals',
                         'uawrite = opcua.tools:uawrite',
                         'uasubscribe = opcua.tools:uasubscribe',
+                        'uahistoryread = opcua.tools:uahistoryread',
```
Comment (oroulet) [inline setup.py]: great! nice to finally get som history supprt

File: opcua/client.py (hunk 2 above)
Comment (oroulet) [inline opcua/client.py]: Then you can add a simple history_read(node, start_time, end_time) to Client if you think it is usefull

**Comment by alkor** [comment]
@oroulet Hmm, yes, I'll move history functions to Node class.

File: opcua/client.py (hunk 2 above)
Comment (oroulet) [inline opcua/client.py]: maybe method in Client should allow to read several nodes at once.... you decide ....

**Comment by alkor** [comment]
@oroulet I moved these methods to Node and removed history_read from Client.

**Pull request merged**